ios/RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -881,6 +893,13 @@ - (long long) getFirstInstallTime {
     return [@(floor([installDate timeIntervalSince1970] * 1000)) longLongValue];
 }
 
+#pragma mark - CLLocationManagerDelegate
+
+- (void)locationManager:(CLLocationManager *)manager didChangeAuthorizationStatus:(CLAuthorizationStatus)status
```

Comment:
did a quick scan of stackoverflow and this one looks interesting! What was the behavior of the callback like in real life? Did it get a callback immediately? Was it correct (thus indicating threading is correct...) value on that first call? Subsequent calls? Curious what your results were

https://search.brave.com/search?q=didChangeAuthorizationStatus+CLAuthorizationStatus